add build, install esy, improve English (#472)
marigold-dev/deku: README.md

# Marigold Sidechain

> The Marigold Sidechain to Tezos is designed to increase throughput.

Reviewer: ```suggestion
A sidechain to Tezos by Marigold focused on higher throughput.
```
I apologize for my very confusing language, but the original sentence fragment actually made more sense than the proposed complete sentence.

Author: Gotcha, I will keep the original then. Thanks.
